### DISCUSSION TITLE
SQL Server: Select * from <table_name> and "Edit Procedure"

### DISCUSSION THREAD

**Discussion by LRHF** [dbeaver/dbeaver#14323] [Q&A]
I use SqlDbx frequently (within a SQL Sever DB), and now I'm exploring DBeaver. There are a couple of features I would like to see in DBeaver.

1. When you right click over a table name (in the SQL editor), the first line of the contextual menu says: "select * from <table_name>". This is really useful: 
![image](https://user-images.githubusercontent.com/22406500/75199599-6521a180-57c8-11ea-96c3-dcefe659d334.png)

2. "Edit Procedure" option is directly accessible in the contextual menu: 
![image](https://user-images.githubusercontent.com/22406500/75199726-b29e0e80-57c8-11ea-94b0-cbe1b461f670.png)

Thanks!

**Comment by uslss** [comment]
thanks for suggestion

**Comment by p91paul** [comment]
for 1, try ctrl+click on the table, it should open table editor

**Comment by LRHF** [comment]
> for 1, try ctrl+click on the table, it should open table editor
Thanks @p91paul, I didn't know that feature
